File: verba-data/src/main/java/ru/nsu/ccfit/verba/verbadata/platform/exeption/NotFoundException.java
package ru.nsu.ccfit.verba.verbadata.platform.exeption;


import ru.nsu.ccfit.verba.verbadata.platform.enumeration.Code;
import ru.nsu.ccfit.verba.verbadata.platform.exeption.BasicVerbaException;

public class NotFoundException extends BasicVerbaException {
    public NotFoundException() {
        super(Code.NOT_FOUND);
    }

    public NotFoundException(String message) {
        super(Code.NOT_FOUND, message);
    }
}

File: verba-api/src/main/kotlin/ru/nsu/ccfit/verba/verbaapi/platform/exception/NotFoundException.kt
package ru.nsu.ccfit.verba.verbaapi.platform.exception

import ru.nsu.ccfit.verba.verbaapi.platform.Code


class NotFoundException : BasicVerbaException {
    constructor() : super(Code.NOT_FOUND)
    constructor(message: String) : super(Code.NOT_FOUND, message)
}
